bin/mcb

```diff
@@ -36,6 +37,11 @@ $mcb = Cri::Command.define do
     puts cmd.help
     exit 0
   end
+
+  option :c, :config, 'use provided config file',
```

Comment:
can you throw in an example config file

Reply:
Not really meant to be used this way, as in ... we shouldn't have to add the config stuff by hand. mcb commands will use this to save stuff, and if the user has to do something special they can prompt the user to do so (e.x. the new audit command will [ask the user to add their email](https://github.com/DFE-Digital/manage-courses-backend/pull/324/files#diff-fabca55fa119b555a7942669c431eafeR212))